src/elements/train/train/train.ts

```diff
@@ -43,7 +43,8 @@ class SbbTrainElement extends SbbNamedSlotListMixin<
 
   /** General label for "driving direction". */
   @forceType()
-  @property({ attribute: 'direction-label' })
+  // Has to be reflected as it is used in core.scss as selector.
```

Comment:
We don't generally write a comment about this. In my eyes the presence of the reflect automatically justifies this.
Or is there a reason you want to highlight it?